tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/error/ErrorTest.java

```diff
@@ -415,6 +415,22 @@ public void testStackTraceWithErrorCauseLocation() {
                 "\t   error_test:testStackTraceWithErrorCauseLocation(error_test.bal:466)");
     }
 
+    @Test
+    public void testErrorStackTrace() {
+        Exception expectedException = null;
+        try {
+            BRunUtil.invoke(errorTestResult, "testErrorStackTrace");
+        } catch (Exception e) {
+            expectedException = e;
+        }
+
+        Assert.assertNotNull(expectedException);
```

Comment:
Shall we move this assertion to bal code.

Reply:
Shall we add a testw ith interops as well.

    void interop() {
foo()
} 
void foo() {
throw Errorcreator.createerrorVale();
}

Reply:
need to have multiple java method levels

Reply:
Addressed with https://github.com/ballerina-platform/ballerina-lang/pull/32156/commits/2b00b4661100f5ab3d019fc2d767f6d8fcdf6cb1